api/pkg/apis/v1alpha1/managers/stage/stage-manager_test.go

```diff
@@ -60,7 +60,7 @@ func TestCampaignWithSingleMockStageLoop(t *testing.T) {
 				"test": {
 					Provider: "providers.stage.mock",
 					Inputs: map[string]interface{}{
-						"foo": "${{$output(test,foo)}}",
+						"foo": "${{$if($equal($output(test,foo), 0), $trigger(foo, 0), $output(test,foo))}}",
```

Comment:
If $trigger doesn't work, will this unit test still pass?

Reply:
Yes. It's not appropriate here. Will remove.